### PR TITLE
docs: Added documentation for the project

### LINKED ISSUE #20: Add README and CONTRIBUTING markdowns
Need to add info in README and CONTRIBUTING guidelines.
Started Work #5

### PR DESCRIPTION
**Pull request by yashlamba**
Closes #20 
We need to start working hard on this and tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
-# simulate
\ No newline at end of file
+# Simulate 
+[![Gitter](https://badges.gitter.im/codEd-org/simulate.svg)](https://gitter.im/codEd-org/simulate?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+Collection of simulations and visualizations in a cross platform app made using flutter.
```
Comment (akshansh2000) [inline README.md]: Flutter, capital 'f'?

File: README.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+
+# Contributing
+
+Before contributing, we highly recommend introducing yourself and your ideas on gitter.
+
+Some pre-requisites:
+
+1. Make sure you have flutter installed (https://flutter.dev/docs/get-started/install)
```
Comment (akshansh2000) [inline README.md]: Flutter*

File: README.md [hunk 3]
```diff
@@ -1 +1,16 @@
...
+2. Try installing the development version on your phone and experiment
```
Comment (akshansh2000) [inline README.md]: Try installing and experimenting with the development version on your phone

File: README.md [hunk 4]
```diff
@@ -1 +1,16 @@
...
+3. Don't hesitate in asking questions, we'll be happy to help
+   
+All the further guidelines can be found in the [documentation](docs/CONTRIBUTING.md)
```
Comment (akshansh2000) [inline README.md]: ~the~

File: docs/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# CONTRIBUTING
+
+> Before contributing, we highly recommend introducing yourself and your ideas on gitter.[![Gitter](https://badges.gitter.im/codEd-org/simulate.svg)](https://gitter.im/codEd-org/simulate?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+### Setting Up Simulate:
+
+1. Make sure you have flutter installed (https://flutter.dev/docs/get-started/install)
```
Comment (akshansh2000) [inline docs/CONTRIBUTING.md]: Flutter*

File: docs/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+2. Fork the repository
+3. Clone the forked repository:
+    ```console
+    git clone https://github.com/[YOUR USERNAME]/simulate.git
+    ```
```
Comment (akshansh2000) [inline docs/CONTRIBUTING.md:12]: If you're using `https`,
```console
git clone https://github.com/[YOUR USERNAME]/simulate.git
```
If you're using `ssh`,
```console
git clone git@github.com:[YOUR USERNAME]/simulate.git
```

File: docs/CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+4. Connect a device or run an emulator (Make sure your connected device has enabled USB Debugging and isn't on 'Charge Only' when connected)
```
Comment (akshansh2000) [inline docs/CONTRIBUTING.md]: enabled USB Debugging --> USB Debugging enabled

Doesn't matter if it's on charge only

File: docs/CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+### Setting Up Simulate:
+
+1. Make sure you have flutter installed (https://flutter.dev/docs/get-started/install)
+2. Fork the repository
+3. Clone the forked repository:
+    ```console
+    git clone https://github.com/[YOUR USERNAME]/simulate.git
+    ```
+4. Connect a device or run an emulator (Make sure your connected device has enabled USB Debugging and isn't on 'Charge Only' when connected) 
+5. Run:
```
Comment (akshansh2000) [inline docs/CONTRIBUTING.md]: Navigate to the project directory and run:

File: docs/CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+1. Open an issue or report on gitter
+2. Make a new branch
+```console
+git checkout -b [BRANCH NAME]
+```
+3. Fix the bug and open a pull request
+
+#### Adding a new Simulation:
+
+> (We aren't accepting new simulation right now, you can request one by opening an issue or on gitter)
```
Comment (akshansh2000) [inline docs/CONTRIBUTING.md]: simulations*

**Review by akshansh2000 — COMMENTED** [review]
Nice! Made a review inline

File: docs/CONTRIBUTING.md (hunk 6 above)
Comment (yashlamba) [reply docs/CONTRIBUTING.md:12]: I am going with the default thing

File: docs/CONTRIBUTING.md (hunk 7 above)
Comment (yashlamba) [reply docs/CONTRIBUTING.md]: ?

Comment (akshansh2000) [reply docs/CONTRIBUTING.md]: It won't matter if it's on charge only. The app would still run.

Comment (akshansh2000) [reply docs/CONTRIBUTING.md]: And also, change "Make sure your connected device has enabled USB Debugging" to "Make sure your connected device has USB Debugging enabled"

**Review by akshansh2000 — APPROVED** [review]

**Pull request merged**